spectrochempy/analysis/mcrals.py

```diff
@@ -654,15 +774,18 @@ def _Y_preprocessed_change(self, change):
                 self.closureTarget = self.closureTarget
                 self.hardC_to_C_idx = self.hardC_to_C_idx
                 self.hardSt_to_St_idx = self.hardSt_to_St_idx
+                self.solverConc = self.solverConc
```

Comment:
There is no custom validation process for these parameters. So this is not needed.

remove:

    self.solverConc = self.solverConc
    self.solverSpec = self.solverSpec

Reply:
done